cmdLine/src/main/scala/topgun/cmdline/FileParser.scala

```diff
@@ -12,23 +18,41 @@ class FileParser(file: File, cmdLine: JfrParseCommandLine, totals: Totals, confi
 
   import configuration.{includeStack, includeThread}
 
-
+  var classLoaderKey = 0
 
   def parse(): Unit = {
 
     println(s"*** File $file")
     var allocationInNewTlab: Boolean = false
     var allocationOutsideTlab: Boolean = false
     var methodProfilingSample: Boolean = false
+    var classPaths =  mutable.ArraySeq[String]()
+
+    def extractClassPath(event: RecordedEvent): Unit = {
+      event.getFields.forEach(
+        x => {
+          if(event.getValue("key").equals("java.class.path")) {
+            val clazzpaths:String = event.getValue("value")
+            val delim = if (System.getProperty("os.name").toLowerCase.contains("win")) ";" else ":"
```

Comment:
we need the os.name where the recording took place, not the one where we are reading the file

Reply:
oh yes, It total skipped me. thanks. I fix that now